checks.d/http_json.py

```diff
@@ -0,0 +1,56 @@
+from checks import AgentCheck
+import urllib2
+import json
+
+class HTTPJSONCheck(AgentCheck):
+    def check(self, instance):
+        if 'url' not in instance:
+            self.log.info("Skipping instance, no url found.")
+            return
+
+        default_timeout = self.init_config.get('default_timeout', 5)
+        timeout = float(instance.get('timeout', default_timeout))
+
+        tags = instance.get('tags', None)
```

Comment:
You don't need the , None part. 
By default .get() will return None if not found.